css/class-naming-conventions/readme.md

```diff
@@ -123,21 +123,80 @@ Sometimes you'll write a modifier for a component and you want that modifier to
 </div>
 ```
 
+
+#### Nested inside the component
```

Comment:
Would it be clearer if the headings for the different methods were numbered 1, 2, 3?